options/App.jsx

```diff
@@ -4,31 +4,65 @@ import WebContent from "./components/WebContnet";
 import { WebSearchContext } from "./context/WebSearchContext";
 
 function App() {
-  const [searchValue, setSearchValue] = useState("");
   const [searchKeyword, setSearchKeyword] = useState("");
-  const [urlNewList, setUrlNewList] = useState([]);
+  const [reSearchKeyword, setReSearchKeyword] = useState(searchKeyword);
 
-  useEffect(() => {
-    chrome.storage.local.get(["initialSearchValue"], (result) => {
-      if (result.initialSearchValue) {
-        setSearchValue(result.initialSearchValue);
-      }
+  const [filteredData, setFilteredData] = useState({ data: [] });
+  const [sortedHistory, setSortedHistory] = useState([]);
+
+  const fetchStorageData = () => {
+    chrome.storage.local.get(null, (items) => {
+      let overallMaxTimestamp = -Infinity;
+      let latestItem = null;
+      const historyArray = [];
+
+      Object.entries(items).forEach(([keyword, data]) => {
+        if (!data || data.length === 0) return;
+
+        const maxTimestampForCategory = data.reduce((max, entry) => {
+          const timestamp = Object.values(entry)[0].timestamp;
+          return timestamp > max ? timestamp : max;
+        }, -Infinity);
+
+        const item = { keyword, data, maxTimestamp: maxTimestampForCategory };
+        historyArray.push(item);
+
+        if (maxTimestampForCategory > overallMaxTimestamp) {
+          overallMaxTimestamp = maxTimestampForCategory;
+          latestItem = item;
+          setSearchKeyword(`${keyword}`);
```

Comment:
```suggestion
          setSearchKeyword(`${keyword}`);
```
깔끔하게 잘 작성하신 코드 같습니다. 수고하셨습니다.
이 부분 keyword가 이미 문자열일 것 같은데, 왜 템플릿 리터럴을 사용하셨는지 궁금합니다.
제 생각에는 만약 keyword에 null 이나 undefined가 들어있을 때 문자열로 변경되어 들어갈 수 있다고 생각이 들어서 질문드리게 되었습니다.

Reply:
문자열 keyword를 가져오려고 의도한게 맞습니다. 이미 문자열인 상태일텐데 객체의 프로퍼티명이 이미 문자열임에도 불구하고 객체자체에 꼿혀 템플릿 리터럴로 문자열 변환해주어야 한다고 생각해버렸네요. 
체크해주신 대로 템플릿 리터럴을 제거하였으며 로직도 이상 없이 동작하는 부분 확인하였습니다. 
질문 및 로직 확인 감사합니다.